src/parseTools.mjs

```diff
@@ -79,6 +79,13 @@ export function preprocess(filename) {
       .replace(/\bimport\.meta\b/g, 'EMSCRIPTEN$IMPORT$META')
       .replace(/\bawait import\b/g, 'EMSCRIPTEN$AWAIT$IMPORT');
   }
+  if (USE_CLOSURE_COMPILER && AUDIO_WORKLET) {
+    // Closure compiler doesn't currently handle static getters.
+    text = text.replace(
```

Comment:
As far as I can tell, that property is not used anywhere, so a simpler fix might be to just remove it (which also gives a tiny code size win).

Reply:
This property is part of the audio worklet API.. the system reads it.  without this fix the audio worklet tests fail.